pharos-cluster.gemspec

```
@@ -3,6 +3,13 @@ lib = File.expand_path("../lib", __FILE__)
 $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 require "pharos/version"
 
+files = Dir['README', 'LICENSE', 'licenses/*', 'bin/*', 'lib/**/*', 'addons/**/*']
+require_paths = ['lib']
+if ENV['PHAROS_NON_OSS'].to_s == 'true'
```

```suggestion
if ENV['PHAROS_NON_OSS'] == 'true'
```

`nil == anything` is false